<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowViewUI</class>
 <widget class="QMainWindow" name="MainWindowViewUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(11, 0, 57); color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
      </property>
      <property name="text">
       <string>Oversight of current products in the making</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="buttons_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(64, 102, 72);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QPushButton" name="distribute_wf_btn">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;
</string>
         </property>
         <property name="text">
          <string>Distribute Workflow</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 58, 172);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="mode_widget_3">
            <property name="styleSheet">
             <string notr="true">font-size: 12pt;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Modes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="distributeTask">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
            </property>
            <property name="text">
             <string>Distribute task</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="changeOrder">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
            </property>
            <property name="text">
             <string>Change Order</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteOrder">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
            </property>
            <property name="text">
             <string>Delete Order</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="createWorkflow">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
         </property>
         <property name="text">
          <string>Create Workflow</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="standardOrder2">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
         </property>
         <property name="text">
          <string>Standard Order</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="newOrder">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:10pt;</string>
         </property>
         <property name="text">
          <string>New Order</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="feedback_mw">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>60</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(221, 255, 0); font-size: 8pt; color: rgb(180, 8, 8);</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="usertype_label">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(4, 255, 0);color:rgb(117, 2, 2);font-size:12pt;</string>
      </property>
      <property name="text">
       <string>Operator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);font-size:11pt;</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Customer</string>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>80</green>
         <blue>84</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Progress</string>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>80</green>
         <blue>84</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Workflow</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Production Marker 1 (e.g. Color)</string>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>80</green>
         <blue>84</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Production Marker 2 (e.g. Material)</string>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>80</green>
         <blue>84</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Due Date</string>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>80</green>
         <blue>84</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notes</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 49, 50);</string>
    </property>
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionAdministration"/>
    <addaction name="actionBenutzer_einsehen"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdministration">
   <property name="text">
    <string>Administration</string>
   </property>
  </action>
  <action name="actionBenutzer_einsehen">
   <property name="text">
    <string>Benutzer einsehen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
